src/prefect/server/services/loop_service.py

```diff
@@ -72,6 +73,10 @@ async def _on_stop(self) -> None:
         self._is_running = False
         self.logger.debug(f"Stopped {self.name}")
 
+    @overload
+    async def start(self) -> NoReturn:
+        ...
+
```

Comment:
I wasn't 100% sure if this is required, but it seemed like a good idea to make sure that there was an overload for the `abc` with the exact parameters (or lack thereof) so we didn't get Liskov substitution warnings about the subclass not implementing the same interface as the base.

Reply:
This isn't needed because `start` in this class has a default value for `loops`, so `pyright` doesn't raise any errors.

Reply:
Ah okay I'll drop it for simplicity